e2e-tests/portal/cypress/e2e/journeys/admin/create-user/create-a-user.spec.js

```diff
@@ -25,6 +25,16 @@ context('Admin user creates a new user', () => {
     roles: ['maker'],
   };
 
+  const userWithInjection = {
+    username: '"$gt": ""',
+    email: '"$gt": ""',
```

Comment:
Can you please change this to `{ "$gt": "" }`

Reply:
changed